omnibus/resources/agent/msi/cal/usercreate.cpp

```diff
@@ -250,7 +250,17 @@ int doCreateUser(const std::wstring& name, const wchar_t * domain, std::wstring&
 
 }
 
-
+int getUserHomeDirectory(const wchar_t* name, std::wstring &path)
+{
+    DWORD ret = 0;
+    USER_INFO_2 *ui = NULL;
+    ret = NetUserGetInfo(NULL, name, 2, (LPBYTE*) &ui);
```

Comment:
From https://docs.microsoft.com/en-us/windows/win32/api/lmaccess/nf-lmaccess-netusergetinfo,

> If you call this function on a domain controller that is running Active Directory, access is allowed or denied based on the access control list (ACL) for the securable object.

Can it create an issue?